kpi/views/tmp_browsers_kpi_2023.view.lkml

```diff
@@ -0,0 +1,116 @@
+view: tmp_browser_kpis_2023 {
+  derived_table: {
+    sql:
+      WITH
+      desktop_dau AS (
+      SELECT submission_date,
+        'DAU' AS metric,
+        'desktop' AS product,
+        COUNTIF(
+          active_hours_sum > 0 AND
+          COALESCE(
+            scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+            scalar_parent_browser_engagement_total_uri_count_sum,
+            0
+          ) > 0
+        ) AS value
+      FROM `mozdata.telemetry.clients_daily`
+      WHERE submission_date >= '2022-12-16'
+      GROUP BY 1
+      ),
+
+      desktop_dau_28_ma AS (
```

Comment:
moving averages can also be calculated using looker table calculations:  #https://help.looker.com/hc/en-us/articles/4420190523667-Moving-Averages-in-Table-Calculations-Community-

Reply:
Thanks, I didn't know that! This looks like a really helpful pattern in general. In this specific case, I think we want to use pure SQL, even though it's much less elegant. One of the goals of this line of work is to use the same KPI definitions everywhere. In the short term, that will mean providing a SQL version of the query that can be used across the stack (see links below). In the long term, this logic will be implemented upstream in bq-etl so that we can just query the metric directly from some `active_users_aggregates` table. That will make the Metric Hub definitions simpler as well.

- Jira [DS-2680](https://mozilla-hub.atlassian.net/browse/DS-2680)
- [Metric Hub proposal](https://github.com/mozilla/metric-hub/pull/66)

Reply:
I agree.  We could eventually add the moving average column to the active users aggregates views. cc @lucia-vargas-a